.github/workflows/cmake.yml

```diff
@@ -52,25 +55,46 @@ jobs:
         run: |
           npm install --location=global azurite
 
-      - name: Configure CMake (Windows)
+      - name: Configure CMake (Windows x64)
         if: ${{ (matrix.OS == 'windows-latest') }}
-        shell: bash
+        shell: pwsh
         run: |
           # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
           # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
           # Note that we need to point CMake to the vcpkg-toolchain-file
-          cmake -B "${{github.workspace}}/build" -DCMAKE_BUILD_TYPE=${{matrix.build}} -DLIBCZI_BUILD_CZICMD=ON -DLIBCZI_BUILD_CURL_BASED_STREAM=ON -DLIBCZI_BUILD_AZURESDK_BASED_STREAM=ON -DLIBCZI_BUILD_PREFER_EXTERNALPACKAGE_LIBCURL=ON -DCMAKE_TOOLCHAIN_FILE="${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake" -DVCPKG_TARGET_TRIPLET=x64-windows-static
+          cmake -B "${{github.workspace}}/build" -A x64 -DCMAKE_BUILD_TYPE=${{matrix.build}} -DLIBCZI_BUILD_CZICMD=ON -DLIBCZI_BUILD_CURL_BASED_STREAM=ON -DLIBCZI_BUILD_AZURESDK_BASED_STREAM=ON -DLIBCZI_BUILD_PREFER_EXTERNALPACKAGE_LIBCURL=ON -DCMAKE_TOOLCHAIN_FILE="$env:VCPKG_INSTALLATION_ROOT/scripts/buildsystems/vcpkg.cmake" -DVCPKG_TARGET_TRIPLET=x64-windows-static
+
+      - name: Configure CMake (Windows ARM64)
+        if: ${{ (matrix.OS == 'windows-latest') }}
+        shell: pwsh
+        run: |
+          # Configure for a cross-compilation to ARM64.
+          # Note that since we are doing a cross compilation, we cannot execute code in order to determine platform
+          # characteristics like endianess and whether unaligned access is allowed. Therefore, we need to set the following
+          # variables manually: CRASH_ON_UNALIGNED_ACCESS=OFF and IS_BIG_ENDIAN=FALSE.
```

Comment:
Maybe also add a link to the triplet documentation or so (https://learn.microsoft.com/en-us/vcpkg/concepts/triplets)?
At least for me it took a few minutes to understand whats going on (after giving up on searching for the toolchain file in the PR 😅). Up to you of course, I would love to read a little hint comment though.

Reply:
I added a link to the page you mentioned (with the x64-build above).